Release 1.16.0 (#4405)
redhat-developer/vscode-openshift-tools: CHANGELOG.md

# Change Log

## 1.16.0 (September 3, 2024)

>  * [#4151](https://github.com/redhat-developer/vscode-openshift-tools/issues/4151) Create a deployment using a BuildConfig

Reviewer: ```suggestion
 * [#4151](https://github.com/redhat-developer/vscode-openshift-tools/issues/4151) Added support for creating deployments using a BuildConfig.
```

Author: Fixed

>  * [#4289](https://github.com/redhat-developer/vscode-openshift-tools/issues/4289) Fixed slow loading of the Applicatoin Explorer

Reviewer 2: Application

Author: Fixed

>  * [#4344](https://github.com/redhat-developer/vscode-openshift-tools/issues/4344) Alizer is introduced to replace "odo analyze" command

Reviewer 2: I would probably lower the alizer migration on the list. It's something that to end-users should be a noop.

Otherwise, everything else looks pretty good!

Author: ACK

>  * [#4108](https://github.com/redhat-developer/vscode-openshift-tools/issues/4108) Introduced the devfile for .NET 9

Reviewer: ```suggestion
 * [#4108](https://github.com/redhat-developer/vscode-openshift-tools/issues/4108) Introduced support for the devfile for .NET 9.

```

Author: Fixed.

Note: Normally we no ending dots at the end of the list items.  If you want it, I can add the "ending dots"

>  * [#4357](https://github.com/redhat-developer/vscode-openshift-tools/pull/4357) Allow commands to work with .yml extension

Reviewer 2: I would make it `.yml` (with the back ticks)

Author: ACK

>  * [#4357](https://github.com/redhat-developer/vscode-openshift-tools/pull/4357) Allow commands to work with `.yml` extension

Reviewer: ```suggestion
 * [#4357](https://github.com/redhat-developer/vscode-openshift-tools/pull/4357)  Enabled command compatibility with files using the `.yml` extension.
```

Author: Fixed

>  * [#4344](https://github.com/redhat-developer/vscode-openshift-tools/issues/4344) Alizer is introduced to replace "odo analyze" command

Reviewer: ```suggestion
 * [#4344](https://github.com/redhat-developer/vscode-openshift-tools/issues/4344) Replaced the `odo analyze` command with the new Alizer tool.
```

Author: Fixed

>  * [#3911](https://github.com/redhat-developer/vscode-openshift-tools/issues/3911) Improved the workflow for entering a container image URL

Reviewer: ```suggestion
 * [#3911](https://github.com/redhat-developer/vscode-openshift-tools/issues/3911) Enhanced the workflow for entering a container image URL.
```

Author: Fixed

>  * [#4289](https://github.com/redhat-developer/vscode-openshift-tools/issues/4289) Fixed slow loading of the Application Explorer

Reviewer: ```suggestion
 * [#4289](https://github.com/redhat-developer/vscode-openshift-tools/issues/4289) Resolved the slow tree loading issue in the Application Explorer.

```

Author: Fixed

>  * [#4349](https://github.com/redhat-developer/vscode-openshift-tools/issues/4349) Fixed broken Helm Chart installation view

Reviewer: ```suggestion
 * [#4349](https://github.com/redhat-developer/vscode-openshift-tools/issues/4349) Fixed the broken Helm Chart installation view
```

Author: Fixed

>  * [#4298](https://github.com/redhat-developer/vscode-openshift-tools/issues/4298) Fixed duplicate deployment names displaying on tree view

Reviewer: ```suggestion
 * [#4298](https://github.com/redhat-developer/vscode-openshift-tools/issues/4298) Fixed the issue with duplicate deployment names displaying in the tree view.
```

Author: Fixed

>  * [#4253](https://github.com/redhat-developer/vscode-openshift-tools/issues/4253) Fixed missing images on "Add OpenShift Cluster" page

Reviewer: ```suggestion
 * [#4253](https://github.com/redhat-developer/vscode-openshift-tools/issues/4253) Fixed the issue of missing images on the "Add OpenShift Cluster" page
```

Author: Fixed

## 1.15.0 (July 16, 2024)

>  * [#4291](https://github.com/redhat-developer/vscode-openshift-tools/pull/4291) Fix issue opening Routes node in OpenShift explorer

Reviewer: ```suggestion
 * [#4291](https://github.com/redhat-developer/vscode-openshift-tools/pull/4291) Fixed the issue preventing the opening of the Routes node in the OpenShift Explorer.
```

Author: Fixed
